Sources/UIApplication.swift

```diff
@@ -87,12 +86,19 @@ import JNI
 
 private let maxFrameRenderTimeInSeconds = 1.0 / 60.0
 
+// ******************
+// Requires Dispatch
+import Dispatch
```

Comment:
Without this, we need CFRunLoopRun to drain the main queue. This is a major breakthrough for the work at hand.